solutions/src/test/java/ru/sberbank/school/task10/MyScalableConcurrentThreadPoolTest.java

```diff
@@ -0,0 +1,208 @@
+package ru.sberbank.school.task10;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.util.Queue;
+import java.util.concurrent.Callable;
+import java.util.concurrent.FutureTask;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MyScalableConcurrentThreadPoolTest {
+    private MyScalableConcurrentThreadPool pool;
+
+    private void fillScalablePool() {
+        pool = new MyScalableConcurrentThreadPool(4, 10);
+        }
+
+    private void fillFixedPool() {
+        pool = new MyScalableConcurrentThreadPool(2);
+        }
+
+    private void fillTasks() {
+        Runnable runnable = () -> {
+        try {
+        Thread.sleep(50);
+        } catch (InterruptedException e) {
+        e.printStackTrace();
```

Comment:
&#39;catch&#39; child has incorrect indentation level 8, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1487491523>*